pkg/interface/src/views/apps/launch/components/tiles/weather.js

```diff
@@ -136,67 +103,66 @@ export default class WeatherTile extends React.Component {
     let secureCheck;
     let error;
     if (this.state.error === true) {
-      error = <p
-          className="f9 red2 pt1"
-              >Please try again.
-        </p>;
+      error = <p className="f9 red2 pt1">Please try again.</p>;
     }
     if (location.protocol === 'https:') {
       secureCheck = (
-        <a className="black white-d f9 absolute pointer"
-           style={{ right: 8, top: 8 }}
+        <a className="black white-d f9 pointer"
            onClick={() => this.locationSubmit()}>
           Detect ->
         </a>
       );
     }
     return this.renderWrapper(
-      <div className="pa2 w-100 h-100 bg-white bg-gray0-d black white-d ">
+      <Box
+        display='flex'
+        flexDirection='column'
+        justifyContent='space-between'
+        height='100%'
+      >
         <a
-          className="f9 black white-d pointer absolute"
-          style={{ top: 8 }}
+          className="f9 black white-d pointer"
           onClick={() =>
             this.setState({ manualEntry: !this.state.manualEntry })
           }
         >
           &lt;&#45;
         </a>
         {secureCheck}
-        <p className="f9 pt5">
+        <Text mt='auto' pb={1}>
```

Comment:
Seems a little too low now? Should be at the top of the tile but below the arrow I think

<img width="138" alt="Screenshot 2020-10-06 at 3 54 28 PM" src="https://user-images.githubusercontent.com/20846414/95253436-5aae9800-07ec-11eb-80f1-178df1109fd6.png">